.eslintrc.js

```diff
@@ -7,10 +7,25 @@ module.exports = {
   },
   plugins: [
     'eslint-plugin-tsdoc',
+    'eslint-plugin-import',
+    'eslint-plugin-unused-imports'
   ],
   extends: [
     'es/node',
+    'plugin:import/errors',
+    'plugin:import/warnings',
+    'plugin:import/typescript'
```

Comment:
They contain default settings for this plugin: https://github.com/benmosher/eslint-plugin-import#settings
Otherwise we'd have to define them ourselves, which would make our linting file a lot longer.